vmm/src/lib.rs

```diff
@@ -2248,10 +2364,53 @@ impl Vmm {
         }
 
         // Now we can restore the rest of the VM.
-        if let Some(ref mut vm) = self.vm {
-            vm.restore()
-        } else {
-            Err(VmError::VmNotCreated)
+        // PANIC: won't panic, we just checked that the VM is there.
+        self.vm.vm_mut().unwrap().restore()
+    }
+
+    /// Checks the migration result.
+    ///
+    /// This should be called when the migration thread indicated a state
+    /// change. The function parses the callback and either shuts down the VMM
+    /// on success or keeps the VM and the VMM running on migration failure.
```

Comment:
> The function parses the callback ...

Which callback?

Perhaps we have leftover documentation from an earlier version?

Reply:
yes, thanks!!

Reply:
fixed